testdata/go/v3/memcached-operator/Makefile

```diff
@@ -48,7 +48,7 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.29.0
+OPERATOR_SDK_VERSION ?= v1.30.0
```

Comment:
Should this be 1.31.0?

Reply:
This version is taken from the binary that is available locally so I think it would be `1.30.0`.